laravel-app/app/Http/Controllers/ProfileController.php

```diff
@@ -26,32 +23,27 @@ public function __construct(ProfileService $userService)
 
     /**
      * Show the profile page.
-     *
-     * @return View
      */
     public function index(): View
     {
+        /** @var User $user */
         $user = auth()->user();
-        return view('profile.index', compact('user'));
+        $posts = $user->posts;
+        return view('profile.index', compact('user', 'posts'));
     }
 
     /**
      * Show the profile edit page.
-     *
-     * @param User $profile
-     * @return View
      */
-    public function edit($profile): View
+    public function edit(): View
     {
+        /** @var User $user */
         $user = auth()->user();
         return view('profile.edit', compact('user'));
     }
```

Comment:
Did you test the edit method? Because I remember the Edit method requires a $id parameter.

Reply:
yes sir, as of now it was not being used since it will automatically edit the currently logged user